docs/pages/tools/integration/pentaho.md

```diff
@@ -1,112 +1,129 @@
-*Last Update {{ git_revision_date }}* 
+---
+tags: integration, middleware, python, IoT
```

Comment:
I took a stab at tags here, @gstachni -- wide open for any changes that need to be made.

Reply:
this is great. it will be easy for us to add the right tags to each page.

Reply:
@gstachni -- the order of the tags does not matter; the Astra Portal code will sort them. Thanks!